bin/dsn.templates/cpp/types.h.php

```diff
@@ -15,9 +16,6 @@
 //
 // !!! WARNING: not feasible for replicated service yet!!! 
 //
-// # define DSN_NOT_USE_DEFAULT_SERIALIZATION
-
-# ifdef DSN_NOT_USE_DEFAULT_SERIALIZATION
```

Comment:
We'd better add some new macros to indicate which serialization method we use. For example, DSN_USE_DEFAULT..., DSN_USE_THRIFT...., DSN_USE_GPROTO...